src/@types/global.d.ts

```diff
@@ -66,6 +66,15 @@ declare global {
         mxModalWidgetStore: ModalWidgetStore;
     }
 
+    export interface DesktopCapturerSource {
```

Comment:
Could this go in DesktopCapturerSourcePicker.tsx rather than the global types? I think we're keeping the global just for overrides on 3rd party types so it would be good not to start putting general type declarations in there, otherwise it'll fill up with all sorts of things very quickly.